modules/ROOT/partials/module-loading/webpack-dev-dependencies.adoc

```diff
@@ -1,15 +1,15 @@
-{% if is_zip_install == true %}
-1. Add the following development dependencies to the project.
-
-- [`webpack`](https://www.npmjs.com/package/webpack)
-- [`webpack-cli`](https://www.npmjs.com/package/webpack-cli)
-- [`mini-css-extract-plugin`](https://www.npmjs.com/package/mini-css-extract-plugin)
-- [`html-webpack-plugin`](https://www.npmjs.com/package/html-webpack-plugin)
-- [`css-loader`](https://www.npmjs.com/package/css-loader)
+ifeval::[{is_zip_install} == true]
+. Add the following development dependencies to the project.
 
+* [`webpack`](https://www.npmjs.com/package/webpack)
+* [`webpack-cli`](https://www.npmjs.com/package/webpack-cli)
+* [`mini-css-extract-plugin`](https://www.npmjs.com/package/mini-css-extract-plugin)
+* [`html-webpack-plugin`](https://www.npmjs.com/package/html-webpack-plugin)
+* [`css-loader`](https://www.npmjs.com/package/css-loader)
```

Comment:
All of these are using old Markdown link syntax and as such aren't rendering correctly.